CHANGELOG.md

```diff
@@ -1,5 +1,147 @@
 # Versions
 
+## v1.0.0-beta.2
+
+_Jul 30, 2025_
+
+### General changes
+
+- Fix navigator checks and ensure safe platform retrieval (#2273) by @mo36924
+- Prevent `Space` key default on keydown (#2295) by @atomiks
+- Check for `performance` existence on server (#2316) by @atomiks
+
+### Accordion
+
+- Destructure `render` prop (#2280) by @atomiks
+- Fix keyboard interactions with elements in the panel (#2321) by @mj12albert
+- Fix open transitions in Safari/Firefox (#2327) by @atomiks
+
+### Alert Dialog
+
+- Support `ShadowRoot` containers (#2236) by @atomiks
+- Add `forceRender` prop to `Backdrop` part (#2037) by @atomiks
+- Improve outside press behavior with touch input (#2334) by @atomiks
+
+### Checkbox
+
+- Fix focusing form controls with `inputRef` (#2252) by @mj12albert
+
+### Collapsible
+
+- Destructure render prop (#2323) by @atomiks
+- Fix open transitions in Safari/Firefox (#2327) by @atomiks
+
+### Dialog
+
+- Support `ShadowRoot` containers (#2236) by @atomiks
+- Add `forceRender` prop to `Backdrop` part (#2037) by @atomiks
+- Improve outside press behavior with touch input (#2334) by @atomiks
+- Use `click` event for outside press dismissal (#2275) by @atomiks
+
+### Field
+
+- Deregister fields from `Form` when unmounting (#2231) by @mj12albert
+
+### Form
+
+- Deregister fields from `Form` when unmounting (#2231) by @mj12albert
+
+### Menu
+
+- Support `ShadowRoot` containers (#2236) by @atomiks
+- Avoid double `useRenderElement` passes (#2256) by @atomiks
+- Improve outside press behavior with touch input (#2334) by @atomiks
+- Close submenus when focus is lost by shift-tabbing (#2290) by @michaldudak
+
+### Menubar
+
+- Fix triggers role (#2317) by @atomiks
+
+### Meter
+
+- Fix ARIA attributes and update docs (#2267) by @mj12albert
+
+### Navigation Menu
+
+- **Breaking change:** Support inlined nesting.
+  The popup collapses in size when animating out on a `<500px` wide viewport.
```

Comment:
Suggestions for more precise BC information are very welcome. 🙏

Reply:
Thanks. 👌 
I've updated the naming, WDYT?
I've also added the same text and a diff in [your PR](https://github.com/mui/base-ui/pull/2269) description, WDYT?

Reply:
Added small tweak suggestion